meetings/2024-10-23.md

```diff
@@ -0,0 +1,154 @@
+* 
+
+#### Reminder of E2EE meeting on Nov 5th
+
+URL: https://www.w3.org/events/meetings/b277ff65-0aad-425e-bd1d-64758cd4547a/20241022T140000/
+* MDJ: Was supposed to be yesterday, moved to Nov. 5
+
+### Vocab PRs
+
+#### Add solid:proxy to Solid Terms
+
+URL: https://github.com/solid/vocab/pull/94
+* eP: My suggestion for this and the data wallet PR, we shouldn't add tests just because someone has idea for new terms. We should only add terms that are in drafts of the spec. We had some conversations with Sarven about where the requirements should go. He put them in a comment in vocab, I don't see this as appropriate place. We need to follow up over there to move specific requirements over to the approproate draft. If normative should go in a draft not vocab comment.
+
+#### Add solid:Wallet for a Data Wallet
+
+URL: https://github.com/solid/vocab/pull/93
+* eP: This should be used in one of the drafts, and then referenced. All reqiurements will be in relevant draft.
+* TBL: Principle of what you should put on the ontology itself. Push back on ontology should be in draft, when the ontology should be the raw definition of the term. If you use range, domain, foaf:account etc. then you know what those terms looks like. So where some terms of the wallet are well-defined and you can put in things. The idea of outting the normative information in the comment, if it helps explain that is fine - you can always kick through to the draft. If you find yourself writing comments in the turtle file, or the RDF, don't waste your expressive power - instead put them into the ontology e.g. rdfs:comment. Good ontologies have comments about themselves. Put stuff in the terms where it will be useful to someone who comes accross on a random piece of information on the net, which some system is looking at in the future, where they can click on the predicate and see what it means.
+* eP: We can follow up in PR.
+
+
+#### http://www.w3.org/ns/solid/terms#updatesViaStreamingHttp2023
+
+URL: https://communitysolidserver.github.io/CommunitySolidServer/latest/usage/notifications/#streaming-http
+
+* eP: Possibly should have created it in `/ns/solid/notifications` instead of `/ns/solid/terms`. We can eaither leave it under `solid:` or I would PR it as a breaking change for CSS v8.0.0
+* eP: I'm going to make PRs to the StreamingHTTPChannel2023 and solid/vocab once we decide on the final namespace
+* eP: I don't have PR yet, but in my implementation for CSS I have a new IRI for ??? which links to ???. One doesn't have to link to notif. channel since someone can stream straight from HTTP. I used this relation but maybe should have been defined in solid notifs. (`/ns/solid/notifications`) rather than solid terms (`/ns/solid/terms`). There will be CSS v8 which could incorporate that breaking change. 
```

Comment:
```suggestion
* eP: Possibly should have created it in `/ns/solid/notifications` instead of `/ns/solid/terms`. We can leave it under `solid:`, or I could PR it as a breaking change for CSS v8.0.0.
* eP: I'm going to make PRs to the `StreamingHTTPChannel2023` and `solid/vocab` once we decide on the final namespace
* eP: I don't have a PR yet, but in my implementation for CSS, I have a new IRI for ??? which links to ???. One doesn't have to link to notification channel since one can stream straight from HTTP. I used this relation, but maybe should have been defined in Solid notifications (`/ns/solid/notifications`) rather than Solid terms.(`/ns/solid/terms`). There will be CSS v8 which could incorporate that breaking change. 
```